reactive/containerd.py

```diff
@@ -124,14 +126,43 @@ def charm_status():
         status.blocked('Container runtime not available')
 
 
-def merge_custom_registries(custom_registries):
+def update_custom_tls_config(config_directory, registries):
+    """
+    Read registries config and write tls files to disk.
+
+    :param str config_directory: containerd config directory
+    :param List registries: juju config for custom registries
+    :return: None
+    """
+    for registry in registries:
+        for opt in ['ca', 'key', 'cert']:
+            file_b64 = registry.get('%s_file' % opt)
+            if file_b64:
+                try:
+                    file_contents = base64.b64decode(file_b64)
+                except (binascii.Error, TypeError):
+                    log(traceback.format_exc())
+                    log("{}:{} didn't look like base64 data... skipping"
+                        .format(registry['url'], opt))
+                    continue
```

Comment:
```suggestion
                    status.blocked(
                        'Invalid base64 data in custom_registries: {}:{}_file'.format(
                            registry['url'], opt))
                    continue
```

Wadda you think @addyess?  We can still write any remaining keys with valid b64 data, `config_changed` will proceed and write the template with as much data as it has, and maybe things will work just fine.  But at least with the `status.blocked`, operators can easily see that their config isn't quite right without digging through the debug log.

Reply:
i'm OK going into the blocked state.  But if it's blocked -- shouldn't we `exit` here rather than `continue`?  I don't know the juju rules about what to do when you get blocked by config.  Does that mean the charm is stating "I don't like this config... fix me" or "I encountered something i didn't like but did the best i could with what you gave me"